### ISSUE TITLE
Any plans to share the filter configs?

### ISSUE THREAD

**Issue by eu9ene** [hplt-project/OpusCleaner#103]
It would be great to have full examples of the filters that were used to train a language pair. This would simplify the usage of the tool by non-expert users.

**Comment by XapaJIaMnu** [comment]
Yes, you are absolutely correct. We should add it to the readme.

**Comment by kpu** [comment]
git repo of filter configurations?  

@eu9ene the longer term plan is the filters will be displayed as sort of review next to the data sets on Opus.

**Comment by eu9ene** [comment]
> git repo of filter configurations?

I mean the final json files with filters to serve as examples for users

**Comment by kpu** [comment]
Examples are but one use case.  We should be collecting the json files more centrally.

**Comment by jelmervdl** [comment]
This was a plan inside HPLT as well. Jörg [started doing this for OpusFilter](https://github.com/Helsinki-NLP/OpusFilter-hub/tree/main), but OpusCleaner hasn't been long enough stable yet to do this.

I don't have a strong opinion on how to organise it yet. I.e. should we create a general repo, like Jörg's, organised by publication, or just have a single opinionated one, like bergamot-students, and let other people publish theirs on their own Github?

We should publish our internal configurations where we can. It would help with having more examples/documentation and something to assure we're not breaking backward compatibility.
